harmony-signing/README.md

```diff
@@ -0,0 +1,50 @@
+# Harmony Signing Tool
+
+  Harmony releases various excutables (i.e. harmony block chain node code, command line tools etc)  regularly through harmony S3 buckets and open to public. 
+  For better safety, all the released binaries shall be signed by harmony siging key (a RSA 4096 asymmetric key). 
+  The private siging key is maintained by amazon aws KMS(Key Management Service) and the key is not visible by anyone from Harmony.
+
+  Only designated aws users (assigned by harmony aws administrator) with signing permission can use the private key to sign binary. 
+  The public key of harmony signing key is open to public and anyone can use it to verify the authencity of the binary.
+
+
+##  Setup the tools
+
+### Install virtualenv
+```bash
+[sudo] pip install -U setuptools
+[sudo] pip install virtualenv
+```
+
+### Install required packages in virtualenv
+```bash
+python3 -m virtualenv v
+source venv/bin/activate
+pip install -r requirement.txt
+deactivate
+```
+
+## Sign binary before releasing binary (requires AWS account and harmony signing key permission)
+
+  Now you can use the python script sign_binary.py to sign the executable. 
+  Please note that this script has to be invoked inside the virtual environment (source venv/bin/activate).
+  As the script requires the Harmony signing key, please login to your aws account before signing.
```

Comment:
Can we run this script anywhere?  Login to aws account means the credential setup?

Reply:
Yes, it can be run on any machine with python3 and aws cli installed.
To run:
1) run the python3 -m virtualenv venv to create virtual environment
2) install all the depend modules on virtualenv, mainly the AWS python SDK dependents
3) login to aws account, this aws account must have permission to use the signing key
4) run the script